dotCMS/src/main/java/com/dotmarketing/portlets/templates/action/EditTemplateAction.java

```diff
@@ -494,16 +519,19 @@ else if ((cmd != null)
 	 * @throws DotDataException
 	 * @throws DotSecurityException
 	 */
-	private boolean canTemplateBeDeleted (WebAsset template, User user, StringBuilder errorBuilder)
+	private boolean canTemplateBeDeleted (WebAsset template, User user, SessionDialogMessage errorMessage)
 			throws LanguageException, DotDataException, DotSecurityException {
-		List<Contentlet> pages = APILocator.getHTMLPageAssetAPI().findPagesByTemplate((Template)template, user, false);
+		List<Contentlet> pages = APILocator.getHTMLPageAssetAPI().findPagesByTemplate((Template)template, user, false, 10);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)